build/AzureIntegration.targets

```diff
@@ -6,12 +6,8 @@
   <Target Name="BuildAzureIntegration">
     <PropertyGroup>
       <AzureIntegrationProjProperties>
-        AspNetUniverseBuildOffline=true;
         RepoRoot=$(AzureIntegrationProjectRoot);
-        VersionSuffix=$(VersionSuffix);
-        BuildNumberSuffix=$(BuildNumberSuffix);
```

Comment:
Are these properties still available to the Azure integration projects?

Reply:
VersionSuffix should be - that's a built in SDK property. BuildNumberSuffix is being removed, but I didn't see it getting used anywhere in Azure integration.